PERFORMANCE.md

```diff
@@ -0,0 +1,343 @@
+
+#### Inference Standard Run Configuration File
+```bash
+[run]
+name = run-2023-08-17-16:10:12
+path = results/inference-standard-scaling/run-2023-08-17-16:10:12
+smartsim_version = 0.5.0
+smartredis_version = 0.3.1
+db = redis-server
+date = 2023-08-17
+language = ['cpp', 'fortran']
+
+[attributes]
+colocated = 0
+client_per_node = [18]
+client_nodes = [25, 50, 75, 100]
+database_hosts = []
+database_nodes = [4, 8, 16]
+database_cpus = [8]
+database_port = 6780
+batch_size = [1000]
+device = GPU
+num_devices = 1
+iterations = 100
+language = ['cpp', 'fortran']
+db_node_feature = {'constraint': 'P100'}
+node_feature = {'constraint': 'SK48'}
+wall_time = 15:00:00
+```
+
+#### Put Tensor (send image to database)
+![Inference Colo](/figures/put_tensor_inf_std.png "Inference Colocated")
+
+#### Run Script (preprocess image)
+![Inference Colo](/figures/run_script_inf_std.png "Inference Colocated")
+
+#### Run Model (run resnet50 on the image)
+![Inference Colo](/figures/run_model_inf_std.png "Inference Colocated")
+
+#### Unpack Tensor (retrieve the inference result)
+![Inference Colo](/figures/unpack_tensor_inf_std.png "Inference Colocated")
+
+## Colocated Inference
+
+The following are colocated deployment scaling results from the cpp-inference and fortran-inference scaling tests with ResNet-50 and the imagenet dataset. For more information on these scaling tests, please see
+the SmartSim paper on [arXiv](https://arxiv.org/pdf/2104.09355.pdf).
+
+#### Inference Colocated Run Configuration File
+```bash
+[run]
+name = run-2023-08-17-18:23:38
+path = results/inference-colocated-scaling/run-2023-08-17-18:23:38
+smartsim_version = 0.5.0
+smartredis_version = 0.3.1
+db = redis-server
+date = 2023-08-17
+language = ['cpp', 'fortran']
+
+[attributes]
+colocated = 1
+client_per_node = [18]
+client_nodes = [4, 8, 12, 16]
+database_cpus = [8]
+database_port = 6780
+batch_size = [96]
+device = GPU
+num_devices = 1
+iterations = 100
+language = ['cpp', 'fortran']
+node_feature = {'constraint': 'P100'}
+```
+
+#### Put Tensor (send image to database)
+![Inference Colo](/figures/put_tensor_inf_colo.png "Inference Colocated")
+
+#### Run Script (preprocess image)
+![Inference Colo](/figures/run_script_inf_colo.png "Inference Colocated")
+
+#### Run Model (run resnet50 on the image)
+![Inference Colo](/figures/run_model_inf_colo.png "Inference Colocated")
+
+#### Unpack Tensor (retrieve the inference result)
+![Inference Colo](/figures/unpack_tensor_inf_colo.png "Inference Colocated")
+
+## Inference Performance Analysis
+
+In this section, we will compare inference clients: `put-tensor`, `unpack-tensor`, `run_model` and `run_script`
+for colocated and standard deployment.
+
+> Note that Inference is the process of running data points into a machine learning model to calculate an output such as a single numerical score. The SmartSim-Scaling tests use Pytorch's implementation of Resnet50 to feed 
```

Comment:
Delete "Note that".
There is a description of where ResNet-50 came from here: https://datagen.tech/guides/computer-vision/resnet-50/
The current model saver code is no longer using resnet-50, but a different model (even though it's still saved as resnet-50.
I don't think PyTorch implemented resnet-50. We are converting the model to their format, but that's not really an implementation